Linking error for UWP

Current master (681575f) fails on the linking phase for UWP build, related to the IPv6 addition. [Q&A post with the error](https://godotengine.org/qa/9507/error-while-trying-to-compile-godot-for-winrt-windows-phone).

```
tcp_server_winsock.winrt.opt.debug.64.x64.obj : error LNK2001: unresolved external symbol in6addr_any
packet_peer_udp_winsock.winrt.opt.debug.64.x64.obj : error LNK2001: unresolved external symbol in6addr_any
bin\godot.winrt.opt.debug.64.x64.exe : fatal error LNK1120: 1 unresolved externals
scons: *** [bin\godot.winrt.opt.debug.64.x64.exe] Error 1120
scons: building terminated because of errors
```

--- a/drivers/unix/socket_helpers.h
+++ b/drivers/unix/socket_helpers.h
@@ -10,6 +10,10 @@
  #endif
 #endif
 
+#ifdef WINRT_ENABLED
+#define in6addr_any IN6ADDR_ANY_INIT
+#endif
+
 // helpers for sockaddr -> IP_Address and back, should work for posix and winsock. All implementations should use this
 
 static size_t _set_sockaddr(struct sockaddr_storage* p_addr, const IP_Address& p_ip, int p_port) {

